content/plugin/framework/acctests.mdx

```diff
@@ -55,5 +55,32 @@ See the [`TestCase`
 documentation](/plugin/sdkv2/testing/acceptance-tests/testcase) for more
 information on using `resource.TestCase`.
 
--> **Note:** We anticipate releasing an improved testing experience for the
-framework in the near future.
+## Implement id Attribute
+
+In SDKv2, resources and data sources automatically included an implicit, root level `id` attribute. The `id` attribute is no longer implicitly added.
```

Comment:
> The `id` attribute is no longer implicitly added.

Are you referring to the SDK or the framework in this statement?

Reply:
Ick -- the framework. Let's clarify that!

```suggestion
In SDKv2, resources and data sources automatically included an implicit, root level `id` attribute. In framework, the `id` attribute is no longer implicitly added.
```